.github/workflows/deploy-portainer.yaml

```diff
@@ -93,6 +95,7 @@ jobs:
                   cat <<REQUEST > request.json
                   {
                     "Image": "${{ inputs.registry }}/${{ inputs.image_name }}:${{ inputs.tag }}",
+                    "Env": "${{ inputs.env_vars }}",
```

Comment:
`env_vars` is not required, what's the behaviour in case it's not provided ?

Reply:
🤞 I hope it will maintain the current behavior. I'll do some tests before merging 😄